src/client/game.cpp

```diff
@@ -3187,6 +3189,8 @@ PointedThing Game::updatePointedThing(
 			result.intersection_normal.X,
 			result.intersection_normal.Y,
 			result.intersection_normal.Z));
+
+		hud->can_draw_selectionindicator = false;
 	}
```

Comment:
What if an object is pointed and afterwards nothing (ie. toward sky) is pointed?

Reply:
Fixed